api-reference/50 Common/Object Structures/PdfExportGanttProps/dateRange.md

```diff
@@ -8,3 +8,23 @@ type: String | Object
 Specifies the date range for which to export tasks.
 
 ---
+
+The **dateRange** property accepts the following values: 
+
+- *'all'* or *'visible'* to export only visible or entire Gantt data.
+
+        dateRange: 'visible'
+
+- A range of task indexes.
+
+        dateRange: {
+            startIndex: 0,
+            endIndex: 5
+        }
+
+- A range of [dates](https://www.w3schools.com/js/js_date_methods_set.asp).
```

Comment:
- External links should be opened in a new tab.
- A list item doesn't need a dot in the end if it isn't a sentence.
- Do you mean dates or values of Date type?